resources/lib/functions.py

```diff
@@ -653,7 +656,7 @@ def processDirectory(results, progress, params):
 
         # set the item name
         # override with name format string from request
-        if name_format is not None:
+        if (name_format is not None) and (item.get("Type") == "Episode"):
```

Comment:
Okay, I changed this up.  It is far from beautiful but without adding another setting just for this node (which seems silly) best I could come up with.  I can add another setting if you want instead..